feat(profiling): add differential flamegraph toolbar

JonasBa opened the pull request:
Add toolbar to differential flamegraph

![CleanShot 2023-12-05 at 11 29 25@2x](https://github.com/getsentry/sentry/assets/9317857/74619c6a-0d85-485d-b28c-3c86863f995f)

JonasBa committed:
feat(differentialflamegraph): render first differential flamegraph

JonasBa committed:
feat(differentialflamegraph): remove activeProfileIndex from graph constructor

JonasBa committed:
feat(differentialflamegraph): ensure view components only read passed prop data

JonasBa committed:
feat(differentialflamegraph): render first view

JonasBa committed:
feat(profiling): use proper graph colors

JonasBa committed:
feat(profiling): account for alpha scaling in tests

JonasBa committed:
fix(profiling): fix flamegraph signature in tests

JonasBa committed:
feat(profiling): add error and loading state

JonasBa committed:
Merge branch 'master' into jb/feat/differential-flamegraph-colors

JonasBa committed:
feat(profiling): fix unused transaction

JonasBa committed:
Merge branch 'jb/feat/differential-flamegraph-colors' into jb/feat/differential-flamegraph-loading-states

JonasBa committed:
feat(profiling): import styled

JonasBa committed:
feat(profiling): add differential flamegraph toolbar

JonasBa committed:
Merge branch 'master' into jb/feat/differential-flamegraph-toolbar

Zylphrex commented on static/app/components/events/eventStatisticalDetector/eventDifferentialFlamegraph.tsx:
```diff
@@ -82,7 +82,7 @@ export function EventDifferenialFlamegraph(props: EventDifferenialFlamegraphProp
         </FlamegraphStateProvider>
       </FlamegraphThemeProvider>
-    </div>
+    </Fragment>
```
This `Fragment` looks redundant

JonasBa replied:
It looks like it yeah, but it's required. The children prop indentation makes it look off

JonasBa committed:
feat(profiling) add transaction and % change text above toolbar (#61146)

![CleanShot 2023-12-05 at 12 14
25@2x](https://github.com/getsentry/sentry/assets/9317857/65d9a1a8-5cc7-43ec-bd27-1c0f2ef9c2ea)
